generative_ai/batch_predict/gemini_batch_predict_bigquery.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import datetime
+import os
+import time
+
+import vertexai
+from vertexai.preview.batch_prediction import BatchPredictionJob
+
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+timestamp = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S-%f")
+
+# TODO(developer): Add your BigQuery table details here
+output_uri = "bq://storage-samples.generative_ai.gen_ai_batch_prediction.predictions.output"
+
+
+def batch_predict_gemini_createjob(output_uri: str) -> BatchPredictionJob:  # Changed return type
```

Comment:
Lets follow the way we have in `gcs` example.
```suggestion
def batch_predict_gemini_createjob(output_uri: str) -> str:
```

Reply:
updated